test/TesterInternal/HostedTestClusterBase.cs

```diff
@@ -27,33 +27,4 @@ public virtual void Dispose()
             HostedCluster?.StopAllSilos();
         }
     }
-
-    public abstract class HostedTestClusterPerTest : OrleansTestingBase, IDisposable
-    {
-        protected TestCluster HostedCluster { get; private set; }
-
-        public HostedTestClusterPerTest()
-        {
-            TestDefaultConfiguration.InitializeDefaults();
-
```

Comment:
this is the one difference between HosterTestClusterPerTest and TestClusterPerTest 

TestDefaultConfiguration.InitializeDefaults(); is never called in TestClusterPerTest 

since its static ```
method 
        static TestClusterPerTest()
        {
            TestDefaultConfiguration.InitializeDefaults();
        }
```
Now we are not initializing this default configuration for tests using HostedTestClusterPerTest by delete it and replace it 

Maybe some explanation will help